Softeq.XToolkit.WhiteLabel.Droid/FragmentBase.cs

```diff
@@ -69,11 +73,24 @@ protected virtual void RestoreViewModelIfNeeded(Bundle? savedInstanceState)
 
         protected virtual void DoAttachBindings()
         {
+            var commands = SetCommands();
+            if (commands != null)
+            {
+                _commandsSubscriptions.AddRange(commands);
+            }
         }
 
         protected virtual void DoDetachBindings()
         {
             this.DetachBindings();
+
+            _commandsSubscriptions.Apply(x => x.Dispose());
+            _commandsSubscriptions.Clear();
+        }
+
+        protected virtual IEnumerable<IDisposable>? SetCommands()
```

Comment:
@sleushunou Let's rename to `SetDisposableCommands` or `SetCommandsWithDisposing`